omnibus/config/projects/chefdk.rb

```diff
@@ -32,6 +32,18 @@
   install_dir "#{default_root}/#{name}"
 end
 
+if windows?
+  override :ruby, version: "2.1.6"
```

Comment:
Don't need 2 if/else branches now.

Just
```
override :ruby, version: "2.1.6"
```

The default devkit is the latest one and it correctly select the platform bit-ness for you.